executor/executor.go

```diff
@@ -21,28 +23,49 @@ import (
 var (
 	command = exec.Command
 	lgr     *logger.Logger
+
+	hasOutputLogFile = false
```

Comment:
does `hasOutputLogFile`  have to be global?

Reply:
I had made it a global variable because it would be easier to manage.
The [flag example](https://golang.org/pkg/flag/) is also set to global,  thus I thought this would be a good way to go.